src/test/java/com/smartsheet/api/models/RowTest.java

```diff
@@ -88,4 +90,17 @@ void testUpdateRowBuilder() {
         assertThat(row.getAbove()).isNull();
     }
 
+    @Test
+    void testJsonObjectExcludesRowIdAfterJsonIgnore() throws Exception {
+        Row row = new Row();
+        row.setId(1L);
+
+        JsonSerializer serializer = new JacksonJsonSerializer();
+        String json = serializer.serialize(row);
```

Comment:
As we are not defining any custom rules (or) reusing the serializer, how about directly using it as follows
```suggestion
        String json = new JacksonJsonSerializer().serialize(row);
```